samples/AspNetCore/GrpcServiceSample/Models/Transaction.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace GrpcServiceSample.Models
+{
+    using System.ComponentModel.DataAnnotations;
```

Comment:
The new sample code has a mix of usings inside the namespace and outside, it would be best to be consistent. IMO outside is better and much more commonly used. (there's a semantic difference, it's not just style).

Reply:
fixed